class/defaults.yml

```diff
@@ -17,3 +17,4 @@ parameters:
           nodeSelector:
             matchLabels:
               node-role.kubernetes.io/infra: ""
+    ingressControllerAnnotations: {}
```

Comment:
Why not make it part of the `ingressControllers.[name]` key, e.g. `ingressControllers.prod.annotations` ?

Reply:
I think this would only add confusion since the key is already used to hold the spec field directly. If we now add annotations on the same level as the values of the spec that would be, at least for me, quite confusing.

Reply:
It wouldn't just add confusion, it would need extra work, since the value `ingressControllers.[name]` is used to populate `.spec` of the resulting IngressController object without further processing at the moment, cf. line 39.

Reply:
Ah, missed that one.